### ISSUE TITLE
Unnecessary<DCompile> tags being added to generated vcxproj

### ISSUE THREAD

**Issue by ratzlaff** [premake/premake-core#1097]
I am updating the premake repo we are using here at my company and running our internal regression tests for it and are noticing that every single one of our tests are now failing because of extra <DCompile> flags being added by default to a generated vxcproj.

We aren't compiling any D sources, so I am a little surprised that this would just be blindly added to every project that is generated.

**Comment by tdesveauxPKFX** [comment]
Yeah, I had the same issue yesterday. I just removed the d module from Premake `_modules.lua` before building it.

You can also override the `dCompile` function with this:
```
function m.dCompile(cfg)
	if p.config.hasFile(cfg, path.isdfile) then
		p.push('<DCompile>')
		p.callArray(m.elements.dCompile, cfg)
		p.pop('</DCompile>')
	end
end
```

I planned to make a PR later today to fix this.

**Comment by ratzlaff** [comment]
I thought removing the module was a bit extreme. The override looks good though, thanks!
